siriuspy/siriuspy/sofb/main.py

```diff
@@ -0,0 +1,593 @@
+"""Main Module of the program."""
+
+import time as _time
+import logging as _log
+from functools import partial as _part
+from threading import Thread as _Thread
+import numpy as _np
+
+from ..epics import PV as _PV
```

Comment:
same comment. missing one empty separation line here, according to the standard I am using. you guys agree?